packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -85,18 +85,22 @@ function MainContent ({
         justify='start'
       >
         <Tab title='CLASSIFICATIONS'>
-          <BarChart
-            data={stats?.data}
-            dateRange={selectedDateRange}
-            type='count'
-          />
+          <Box width='100%' height='15rem'>
```

Comment:
The BarChart is a Grid, so now that there's not a Grid in the Layout, a parent of BarChart must have a defined `height` to fill to. The BarChart has `size='fill'`.